src/network/httpRequest.js

```diff
@@ -0,0 +1,42 @@
+import fetch from 'node-fetch';
+
+/**
+ * Executes an HTTP request.
+ * @param url {String} The URL we want to request.
+ * @param options {Object} The fetch options.
+ * @return {Object} The response data as a Promise.
+ */
+export function request(url, options) {
+  console.log('url:', url, 'options:', options);
+
+  return fetch(url, options)
+    .then(checkStatusCode)
+    .then(parseResponse);
+}
+
+function checkStatusCode(response) {
+  if (response.status >= 200 && response.status < 300) {
```

Comment:
3xx can be redirect, not an error

Reply:
Node-fetch seems to handle redirects. I'll research, but I think if it does redirect it should be returning the code of the redirect response.